processor/src/errors.rs

```diff
@@ -16,7 +16,9 @@ pub enum ExecutionError {
     DivideByZero(u32),
     DuplicateAdviceKey(Word),
     FailedAssertion(u32),
+    InaccessibleMemoryAddress(u64),
```

Comment:
nit: Would probably rename this to `UninitializedMemoryAddress`.

Reply:
Addressed in f4e857e1